package/com.unity.formats.usd/Runtime/Scripts/IO/Materials/UrpShaderImporter.cs

```diff
@@ -0,0 +1,178 @@
+
```

Comment:
We should add Unity copyright header to new files (same pattern as other files, but 2022 and Unity Technologies)